src/VisualStudio/Core/Def/Implementation/Preview/PreviewUpdater.cs

```diff
@@ -41,7 +41,7 @@ public void UpdateView(TextDocument document, SpanChange spanSource)
             var documentText = document.GetTextAsync().Result.ToString();
             if (TextView.TextBuffer.CurrentSnapshot.GetText() != documentText)
             {
-                UpdateBuffer(document, spanSource, out var container, out var documentBackedByTextBuffer);
+                UpdateBuffer(document, spanSource, out _, out _);
```

Comment:
I think it makes sense to enforce named arguments for discarded out arguments. I've seen this many times and I *really* need to know what is being discarded.

Reply:
@alrz That is a good point. I am wondering if we should update the code fix for this rule + also add a separate analyzer/fixer that suggests adding naming arguments for discarded arguments? Probably an addition to https://github.com/dotnet/roslyn/issues/43988?

Reply:
My opinion:  #43988 is too much noise, even though it's "usually" a good idea to use named args, it really actualy depends on the context where you need to add named args for literals. Hardcoding a list of methods doesn't seem to be a solution neither. But since `out _` is actually hiding a piece of info (while saying that this is not being used), named args can be always helpful.

Reply:
One heuristic for #43988 I think is that only offer when literals are mixed with other kinds of expressions. so Add(1) or ConfigureAwait(false) won't get flagged but `M(a, true, b)` does.

Reply:
I have added the suggestions to #43988.